js/views/templates/roomlistview.handlebars

```diff
@@ -43,7 +43,9 @@
 				<span>{{copyLinkText}}</span>
 			</button>
 		</li>
-		<li><div class="separator"></div></li>
+		<li><div class="separator"></div>
+		<li class="app-navigation-entry-menu-caption">{{notificationCaptionText}}</li>
+		</li>
```

Comment:
li inside li? that seems to be a mistake :thinking:

Reply:
Yeah, It's the li from the separator, gonna fix it now!